### ISSUE TITLE
Switch states not visible in High Contrast Mode

### ISSUE THREAD

**Issue by softwarea** [twbs/bootstrap#33359]
Bootstrap 4.6:
The states of a simple switch are not visible in Windows High Contrast Mode.

I used the following code from the documentation and enabled the High Contrast Mode with [Left-Alt]+[Left-Shift]+[Print].
```
<div class="custom-control custom-switch">
  <input type="checkbox" class="custom-control-input" id="customSwitch1">
  <label class="custom-control-label" for="customSwitch1">Toggle this switch element</label>
</div>
```
The toggle indicator is not visible.

Here is a [CodePen](https://codepen.io/softwarea/pen/YzpgjRx) sample.

I believe this is a bug? Anybody knows a quick workaround?

**Comment by alpadev** [comment]
You're right that this needs to be addressed. As a workaround, to make color adjustments work you either have to use `forced-color-adjust: none;` (not really [supported](https://caniuse.com/?search=forced-color-adjust)) or use valid colors from the system palette https://www.w3.org/TR/css-color-4/#css-system-colors.

Leaving this as a reference here: https://blogs.windows.com/msedgedev/2020/09/17/styling-for-windows-high-contrast-with-new-standards-for-forced-colors/

**Comment by softwarea** [comment]
Ok, thank you alpadev. I wasn' sure if this is a Bootstrap bug or Browser issue. Do you think this will be fixed in 4.6? Or in 5.x?

**Comment by alpadev** [comment]
@softwarea not sure about the roadmap. Pinging https://github.com/orgs/twbs/teams/css-review

**Comment by alpadev** [comment]
@softwarea not sure about the roadmap. cc @XhmikosR @patrickhlauke

**Comment by XhmikosR** [comment]
Personally, this is not something I can fix on v4. PRs are welcome if someone wants to fix the issue and doesn't cause any regressions, but v4 needs to be as stable as possible at this point.

**Comment by ffoodd** [comment]
Would you mind providing screenshots, please? Could help to see what happens, for non Windows user. :)

Using system colors could help but should be scoped to high-contrast media queries, and support is not so great. Workaround would be using `border`s alongside `background`—which is the solution [suggested by Adrian Roselli on his "Under-engineered toggles" post](https://adrianroselli.com/2019/03/under-engineered-toggles.html) (in which he also enhances support using the high-contrast media query, too).

**Comment by alpadev** [comment]
@ffoodd 

![image](https://user-images.githubusercontent.com/2838324/111774653-0d147c00-88b0-11eb-9078-817be0288042.png)

![image](https://user-images.githubusercontent.com/2838324/111774665-0f76d600-88b0-11eb-8279-294472eb51c8.png)
